_includes/asg/ebs-template.md

```diff
@@ -6,21 +6,24 @@ For PX 1.3 and higher, you can specify a template spec which will be used by Por
 
 The spec follows the following format:
 ```
-"type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>"
+"type=<EBS volume type>,size=<size of EBS volume>,iops=<IOPS value>,enc=<true/false>,kms=<CMK>"
 ```
 
 * __type__: Following two types are supported
     * _gp2_
     * _io1_ (For io1 volumes specifying the iops value is mandatory.)
 * __size__: This is the size of the EBS volume in GB
 * __iops__: This is the required IOs per second from the EBS volume.
+* __enc__:  This needs to be set to true if EBS volumes need to be encrypted. Default: false
```

Comment:
Can you create a ticket in PSG project and assign to binu. Include details on how the AWS ASG page should look on install.portworx.com.

Reply:
https://portworx.atlassian.net/projects/PSG/issues/PSG-24